src/components/Chat/ChatButton.jsx

```diff
@@ -0,0 +1,89 @@
+import { 
+  SendBirdProvider, 
+  withSendBird,
+  sendBirdSelectors, 
+} from 'sendbird-uikit';
+import 'sendbird-uikit/dist/index.css';
+import {React, useContext} from 'react';
+import ChatContext from './ChatContext'
+
+const appId = process.env.REACT_APP_SENDBIRD_ID
+
+const CustomChannel = ({createChannel, sdk, sendMessage}) => {
+  const props = useContext(ChatContext)
+
+  return (
+    <>
+      <button onClick={() => {
+        let message = prompt("Send an intial message:", "Hi! I am interested in being your client...")
+        console.log(message)
+        console.log(props.pt)
+
+        let channelParams = new sdk.GroupChannelParams()
+        channelParams.isPublic = false
+        channelParams.isEphemeral = false
+        channelParams.isDistinct = true 
+      
+        const ptId = props.pt.id
+        var userIds = [ptId]
```

Comment:
```suggestion
        const userIds = [ptId]
```
Does this need to be a var?

Reply:
nope